docs/source/developer/esi-with-devstack.rst

```diff
@@ -0,0 +1,242 @@
+Deploying Node with DevStack and Ironic
+=======================================
+
+This covers the steps to deploy a node using Ironic and set up DevStack with Ironic integration, including creating fake nodes in Ironic.
```

Comment:
This no longer covers deploying a node, so the scope of the document is creating a development deployment of ESI on DevStack.